lib/ffmpeg/movie.rb

```diff
@@ -141,6 +143,18 @@ def initialize(path)
       end
     end
 
+    def ffprobe_command()
```

Comment:
nit: Should omit `()` when there are no parameters in use (applies to `ffmpeg_command`, too)